### PR TITLE
Productionize CDC sources

### PR DESCRIPTION
**Pull request by jbartok**

### CODE REVIEW

**Comment by jbartok** [comment]
verify

**Comment by jbartok** [comment]
verify

File: cdc/src/main/java/com/hazelcast/jet/contrib/cdc/CdcSources.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        return KafkaConnectSources.connect(properties,
+                record -> {
+                    String keyJson = Values.convertToString(record.keySchema(), record.key());
+                    String valueJson = Values.convertToString(record.valueSchema(), record.value());
+                    return new ChangeEventMongoImpl(keyJson, valueJson);
+                }
+        );
+    }
+
+    private static void checkSet(Properties properties, String key) {
```
Comment (eminn) [inline cdc/src/main/java/com/hazelcast/jet/contrib/cdc/CdcSources.java]: We can replace this with `com.hazelcast.internal.util.Preconditions#checkHasText`

**Comment by jbartok** [comment]
**OBSOLETE**, replaced by [Jet #2142](https://github.com/hazelcast/hazelcast-jet/pull/2142).

**Pull request closed without merging**